LanguageFeatures/Extension-types/static_analysis_member_invocation_A06_t02.dart

```diff
@@ -2,11 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion We say that an extension type declaration DV has a non-extension
-/// type member named n in the case where DV does not declare a member named n,
-/// but DV has a direct extension type superinterface V that has a non-extension
-/// type member named n, or DV has a direct non-extension type superinterface T
-/// whose interface contains a member signature named n.
+/// @assertion We say that an extension type declaration DV has an extension
+/// type member named n in the cases where:
+/// - DV declares a member named n.
+/// - DV has no such declaration, but DV has a direct extension type
+///   superinterface V that has an extension type member named n due to a member
+///   declaration DM2, and DV does not declare a member that precludes DM2.
```

Comment:
In this case the member is indeed a non-extension type member of the extension type, so the `@assertion` should probably be taken from the paragraph right after the one about extension type members which is the current `@assertion`.

Reply:
Please check the other tests for the same issue: The `@assertion` should be about extension type members when the test is about extension type members, and similarly for non-extension type members.

Reply:
Thanks!